diff --git a/modules/modules.nix b/modules/modules.nix
index 0e18795..ecae348 100644
--- a/modules/modules.nix
+++ b/modules/modules.nix
@@ -27,7 +27,7 @@ let
     (loadModule ./misc/gtk.nix { })
     (loadModule ./misc/lib.nix { })
     (loadModule ./misc/news.nix { })
-    (loadModule ./misc/nixpkgs.nix { })
+    # (loadModule ./misc/nixpkgs.nix { })
     (loadModule ./misc/numlock.nix { condition = hostPlatform.isLinux; })
     (loadModule ./misc/pam.nix { })
     (loadModule ./misc/qt.nix { })
@@ -144,8 +144,8 @@ let
     (loadModule ./xcursor.nix { })
     (loadModule ./xresources.nix { })
     (loadModule ./xsession.nix { })
-    (loadModule <nixpkgs/nixos/modules/misc/assertions.nix> { })
-    (loadModule <nixpkgs/nixos/modules/misc/meta.nix> { })
+    (loadModule "${pkgs.path}/nixos/modules/misc/assertions.nix" { })
+    (loadModule "${pkgs.path}/nixos/modules/misc/meta.nix" { })
   ];
 
   modules = map (getAttr "file") (filter (getAttr "condition") allModules);
@@ -155,7 +155,7 @@ let
     config._module.args.pkgs = lib.mkDefault pkgs;
     config._module.check = check;
     config.lib = import ./lib { inherit lib; };
-    config.nixpkgs.system = mkDefault pkgs.system;
+    # config.nixpkgs.system = mkDefault pkgs.system;
   };
 
 in
